### PR TITLE
2019 6 11 cache build

### PR DESCRIPTION
**Pull request by benjaminleighton**
New linksets and datasets for cache build

### CODE REVIEW

File: common/common.env [hunk 1]
```diff
@@ -20,13 +20,18 @@ S3_SOURCE_DATA_PATH=/source-data
 S3_DATASET_PATH=/auto-generated/datasets
 ASGS2016_DATASET_FILE=asgs2016.trig.gz
 ASGS2011_DATASET_FILE=asgs2011.trig.gz
-GEOFABRIC_DATASET_FILE=geofabric.trig.gz
+GEOFABRIC_DATASET_FILE=geofabric2.trig.gz
```
Comment (jyucsiro) [inline common/common.env:23]: should probably remain as geofabric.trig.gz. we might need to remove geofabric2.trig.gz. was waiting to do a diff to see if the outputs were the same to verify the harvest went ok.

File: common/common.env [hunk 2]
```diff
@@ -20,13 +20,18 @@ S3_SOURCE_DATA_PATH=/source-data
...
+GNAF1605_STREET_LOCALITY_DATASET_FILE=gnaf_201605_streetLocality_instances.nt.gz
+GNAF1605_ADDRESS_SITES_DATASET_FILE=gnaf_201605_addressSites_instances.nt.gz
 
 #This is the used to create and consume the linksets 
+S3_MANUAL_LINKSET_PATH=/linksets
 S3_LINKSET_PATH=/auto-generated/linksets
 MB2CC_LINKSET_FILE=ls_mb16cc.ttl
 MB11MB16_LINKSET_FILE=ls_mb11mb16.trig.gz
 ADDR16052CC_LINKSET_FILE=ls_addr201605catch.trig
+ADDR16052MB_LINKSET_FILE=ls_addr1605mb16.ttl.gz
```
Comment (ashleysommer) [inline common/common.env]: why not use ls_addr1605mb16.trig.gz ?

Comment (ashleysommer) [inline common/common.env]: what about ls_addr1605mb11 linkset?

File: common/common.env [hunk 3]
```diff
@@ -33,5 +33,6 @@ S3_LINKSET_PATH=/auto-generated/linksets
 MB2CC_LINKSET_FILE=ls_mb16cc.ttl
 MB11MB16_LINKSET_FILE=ls_mb11mb16.trig.gz
 ADDR16052CC_LINKSET_FILE=ls_addr201605catch.trig
-ADDR16052MB_LINKSET_FILE=ls_addr1605mb16.ttl.gz
+ADDR16052MB16_LINKSET_FILE=ls_addr1605mb16.trig.gz
+ADDR16052MB11_LINKSET_FILE=ls_addr1605mb11.trig.gz
```
Comment (ashleysommer) [inline common/common.env:37]: looks good

**Review by ashleysommer — APPROVED** [review]

**Review by jyucsiro — APPROVED** [review]

**Pull request merged**